Makefile

```diff
@@ -154,7 +156,7 @@ build-ui: ## Build UI
 	(cd web/; pnpm install --frozen-lockfile; pnpm build)
 
 build-server: build-ui ## Build server
-	CGO_ENABLED=1 GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -ldflags '${LDFLAGS}' -o dist/woodpecker-server github.com/woodpecker-ci/woodpecker/cmd/server
+	CGO_ENABLED=0 GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -ldflags '${LDFLAGS}' -o dist/woodpecker-server github.com/woodpecker-ci/woodpecker/cmd/server
```

Comment:
that also would drop sqlite support

Reply:
Well, right now, it's m1 or SQLite. 

If you can find a better way to get m1 and SQLite, I'm all for it. And if you need help testing a change to see if it works on m1, I'm happy to test.

Reply:
we could make it a overwrite-able env, so by default .. and our CI do build with CGO and you still can build on m1

Reply:
Personally, I'd rather it just did the right thing when building on an m1. But my efforts to make it figure it out failed miserably, and this is really not my primary focus.

That said, an env var would be fine w/ me. Can you suggest the code to make that work?